### PR TITLE
feat: added sub duration in new User Card

### LINKED ISSUE #593: [REQUEST] Show sub duration in new user card
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This feature does not exist in the latest nightly version

- [X] I have checked the latest nightly version

### Feature Description

The native twitch user card provides information about the followage and sub duration:
image
The new usercard currently only shows followage:
image

It would be nice if the sub duration and tier could be added to the new user card. (Also display it if the user was previosuly subscribed)

### PR DESCRIPTION
**Pull request by TheRealGI**
Closes #593 

The data was already provided in ``` twitchUserCardQuery.Response ``` however wasn't typed. 

- The sub duration will be displayed when month > 0

**Never subscribed**
sub3

**Currently subscribed**
Sub1

**Was subscribed**
Sub2
**EDIT**
As @ayyybubu  wished I added also the creation date to the User Card
image

### CODE REVIEW

File: src/site/twitch.tv/modules/chat/components/user/UserCard.vue [hunk 1]
```diff
@@ -32,6 +32,19 @@
 						<p v-if="data.targetUser.relationship.followedAt">
 							{{ t("user_card.following_since_date", { date: data.targetUser.relationship.followedAt }) }}
 						</p>
+						<p v-if="data.targetUser.relationship.subscription.months">
+							{{
```
Comment (AnatoleAM) [inline src/site/twitch.tv/modules/chat/components/user/UserCard.vue:36]: For the sake of readability consider using a `v-else` instead

**Comment by ayyybubu** [comment]
Ideally would be nice to bring back account age as well as icons for each line but i guess it's up to Anatole to do this 

aSHcGJo

File: src/site/twitch.tv/modules/chat/components/user/UserCard.vue (hunk 1 above)
Comment (TheRealGI) [reply src/site/twitch.tv/modules/chat/components/user/UserCard.vue:47]: Hope it's more clear now. Also rebased on the most recent master branch

Comment (TheRealGI) [reply src/site/twitch.tv/modules/chat/components/user/UserCard.vue:51]: I added also the creation date to the user card. See top comment. If not in scope I can revert the last commit.

**Review by AnatoleAM — APPROVED** [review]

**Comment by AnatoleAM** [comment]
Thanks for your contributions, as a thank-you for making a number of improvements to the extension, please reply with your twitch username if you'd like to be entitled the "Contributor" badge in chat. :rocket:

**Comment by TheRealGI** [comment]
I appreciate it. Username is **domsking2013** @AnatoleAM

**Comment by Noctumsempra** [comment]
It would be so nice to have the subscribed time when you hover a user's star icon in the chat rather than just "Subscriber".
I never touched Typescript but will try the effort to checkout the repo and see if I even get the part of the code where chat is rendered to see how to attach that information to each subicon in the chat. Wish me luck. How do I do if happen to solve it? PR? (Never did a PR but I will, if everything goes fine, hehe)

**Pull request merged**